builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -326,6 +326,18 @@ def loadServiceConfigurationData() {
             sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/dev-config.json"
             sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/stg-config.json"
             sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/prod-config.json"
+
+            sh "sed -i -- 's/{default_provider}/${config_loader.AWS.DEFAULTS.PROVIDER}/g' ./config/dev-config.json"
```

Comment:
This should be `configLoader.JAZZ.DEFAULTS.PROVIDER`. See https://github.com/tmobile/jazz/pull/650/files#diff-c8c17ed23a513bb61c8d8cee76aa8faeR96

Reply:
done